spec/bibliothecary_spec.rb

```diff
@@ -85,13 +85,13 @@
         :dependencies=>[],
         :kind => 'lockfile',
         success: true,
-        :related_paths=>["Gemfile"]},
+        :related_paths=>["Gemfile", "bibliothecary.gemspec"]},
        {:platform=>"rubygems",
         :path=>"bibliothecary.gemspec",
         :dependencies=>[],
         :kind => 'manifest',
         success: true,
-        :related_paths=>[]}])
+        :related_paths=>["Gemfile.lock"]}])
```

Comment:
I'm wondering if we want Gemfile to show up here too; but maybe figure out how we use it and it might not be necessary

Reply:
It seems like we have manifest->lockfile and lockfile->manifest related paths, but not manifest->manifest, although I see what you're saying. I'll open another pull if I need to.

Reply:
Although as I think about it more, you're probably right. Let me check our usage of this.